src/index.js

```diff
@@ -13,9 +13,7 @@ module.exports = class SentryPlugin {
 		this.projectSlug = options.project
 		this.apiKey = options.apiKey
 
-		this.releaseVersion = _.isFunction(options.release)
-			? options.release()
-			: options.release
+		this.releaseVersion = options.releaseVersion;
```

Comment:
Shoot! Sorry for the sloppiness. In all honesty I was in a bit of a hurry when I made this PR.
I'll go ahead and fix this.